ext/cfx-ui/src/app/servers/connecting-popup.component.html

```diff
@@ -1,4 +1,4 @@
-<div [class.overlay]="!minMode" [class.minmode]="minMode" [class.active]="showOverlay">
+<div [class.overlay]="!minMode" [class.minmode]="minMode" [class.active]="showOverlay" (keydown)="onKeyPress($event)" tabindex="1">
```

Comment:
Does `tabindex` in this case make it automatically take focus when added to the DOM?